cms/static/js/views/paging.js

```diff
@@ -0,0 +1,143 @@
+;(function (define) {
```

Comment:
Is this the original file just being moved back? It would be good to preserve the history, if possible. Maybe this file should be moved back, and then a new common page introduced.